lib/ohai/loader.rb

```diff
@@ -98,7 +99,7 @@ def load_plugin(plugin_path)
     # @return [Object] class object for the ohai plugin defined in the file
     def load_plugin_class(plugin_path)
       # Read the contents of the plugin to understand if it's a V6 or V7 plugin.
-      contents = ""
+      contents = String.new
       begin
         logger.trace("Loading plugin at #{plugin_path}")
         contents << IO.read(plugin_path)
```

Comment:
```suggestion
      contents = begin
        logger.trace("Loading plugin at #{plugin_path}")
        IO.read(plugin_path)
```

I mean, if we're really about saving memory with this PR...

Reply:
Wouldn't we want the trace outside the begin there?

Reply:
Yeah that probably makes more sense 👍